routers/api/v1/repo/issue_label.go

```diff
@@ -173,13 +173,14 @@ func DeleteIssueLabel(ctx *context.APIContext) {
 		return
 	}
 
-	label, err := models.GetLabelByID(ctx.ParamsInt64(":id"))
+	label, exist, err := parseLabel(ctx.Params(":id"), ctx.Repo.Repository.ID)
```

Comment:
Probably should just move this down to `models.GetLabelByIDOrName(name string) (*Label, error)`